resume.yaml

```diff
@@ -1,121 +0,0 @@
-      - Gained experience with real-world workflows and development systems.
-      - Presented results to Uber ATG software engineers.
-  - title: Teaching Assistant - Introduction to Programming
-    company: University of Virginia
-    location: Charlottesville, VA
-    dates: Fall 2017 - May 2020
-    notes:
-      - Assisted hundreds of students with introductory programming concepts in Python.
-      - Created <a href="https://www.youtube.com/playlist?list=PLeih3T8PoRaYXpRwCVUmCEQqzZ51qyAmD">review videos</a> and study material for the students.
-      - Helped manage the course as one of four "Head TAs."
-  - title: Software Engineering Intern
-    company: NASA Langley Research Center
-    location: Hampton, VA
-    dates: May 2018 - May 2019
-    notes:
-      - "Data Management for Airborne Atmospheric Composition Data."
-      - Added HDF file compatibility to NASA’s <a href="https://www-air.larc.nasa.gov/missions/TOLNet/tools/FTScan.zip">TOLNet File Format Scanner and Curtain/Profiles Plotter</a>.
-      - Engineered a <a href="https://www-air.larc.nasa.gov/cgi-bin/Driver.cgi?platform=KORUSAQ/DC8_AIRCRAFT">full-stack web application</a> (JQuery, AJAX, cgi, C++) for atmospheric researchers to create customized data merges.
-      - Presented a working prototype to atmospheric researchers and received design feedback.
-      - Continued as a "casual employee" of Science Systems and Applications, Inc (SSAI) to continue work on the merge tool.
-skills:
-  main languages:
-    - Python
-    - Java
-    - Bash
-    - JavaScript
-    - C++
-    - Groovy / Kotlin
-  main other technologies:
-    - HTML
-    - CSS
-    - Gitlab pipelines
-    - Feature Toggles / LaunchDarkly
-    - Unit testing / mocking
-    - Vim
-    - Git
-    - Kubernetes / Helm
-    - SQL
-    - Docker
-    - Terraform / AWS
-    - Functional programming
-    - React
-  also worked with:
-    - Gradle
-    - Django
-    - JQuery
-    - Prometheus
-    - Argo CD
-    - Google chat bots
-    - Chrome Extensions
-    - Intellij Plugins
-    - Autodesk Inventor
-    - LabVIEW
-books:
-  - Clean Code
-  - The Pragmatic Programmer
-  - The Unicorn project
-projects:
-  - name: This Resume
-    dates: February 2022
-    link: "https://github.com/sgatewood/resume"
-    notes:
-      - I didn't have Microsoft Word anymore, so I wrote this in YAML and rendered it into the document you are reading. :-)
-  - name: Capstone Research
-    dates: Fall 2019 - Spring 2020
-    notes:
-      - Worked on a team to create an "automated advising assistant" (Django/React app) for UVA students.
-      - Would have helped students come up with a long-term course plan based on a weighted scheduling algorithm
-      - "Sub-team: Full-Stack/Applications Team"
-  - name: TA sorter
-    dates: Spring 2019
-    link: https://github.com/sgatewood/TA_Sorter
-    notes:
-      - Created a program to help us sort out TA office hours and lab preferences using Max Flow. (Code quality = just for fun...)
```

Comment:
`Created a TA office hours scheduler and lab preferences using Max Flow.` They probably don't have the time to go review this code. You could stick something in the README if you're worried about it, but for the most part, other engineers are understanding that pet projects let code quality slide when the program is basically a fancy POC